app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_size_welcome_text">34sp</dimen>
+    <dimen name="margin_top_35dp">35dp</dimen>
+    <dimen name="margin_top_18dp">18dp</dimen>
+    <dimen name="margin_left_10dp">10dp</dimen>
+    <dimen name="text_size_selection_24sp">24sp</dimen>
```

Comment:
pls use this when you name a resource 
https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Reply:
Done!